docs/guides/guide-for-latex-users.md

```diff
@@ -562,6 +562,19 @@ The example below
 This should be a good starting point! If you want to go further, why not create
 a reusable template?
 
+## Bibliographies { #bibliographies}
+The bibliography system has many benefits. Those include the following:
+- **Inbuilt citation styles**. Typst has over 80 included citation styles
+- **Compability with CSL styles**. Typst is compatibly with
+  [CSL](https://github.com/citation-style-language/styles) bibliography styles.
+- **Support for BibTex**. Typst also supports using BibTex files which previous
```

Comment:
```suggestion
- **Support for BibTeX**. Typst also supports using BibTeX files which previous
```

Ref: [BibTeX - TeX Users Group](https://tug.org/bibtex/)

In addition, I suggest `latex` → `LaTeX`.